### ISSUE TITLE
SNS topic instead of DDS API Call

### ISSUE THREAD

**Issue by aray-wellcome** [wellcomecollection/goobi-infrastructure#437]
I'm not sure if you all have access to this ticket but this is the one I'm referencing https://github.com/wellcomecollection/platform/issues/5494

**Comment by aray-wellcome** [comment]
@alexwlchan Can you help with this? Or point us to someone who can?

**Comment by alexwlchan** [comment]
You’ll need to create new topics in Terraform, and add permissions for Goobi to publish them. Ideally one topic for each of staging/prod.

**Comment by mgeerdsen** [comment]
@alexwlchan So the idea is for the topic to be set up in our environment (one for staging, one for prod) and you will subscribe to it?

**Comment by alexwlchan** [comment]
> So the idea is for the topic to be set up in our environment (one for staging, one for prod) and you will subscribe to it?

Yup, although it'll be DLCS/iiif-builder that subscribes to it, not me.

**Comment by mgeerdsen** [comment]
@alexwlchan Could you let me know the account ID that will be used for the subscription to these topics?

**Comment by mgeerdsen** [comment]
We can now publish SNS notifications from staging Goobi workflow to the topic `digitised-bag-notifications-workflow-staging`  as defined in PR #446 

This still requieres cross account subscription and we need to check that the content of the message is correct.

**Comment by alexwlchan** [comment]
> Could you let me know the account ID that will be used for the subscription to these topics?

It’ll be the Digirati account, which has ID 653428163053

**Comment by mgeerdsen** [comment]
Thanks
The Digirati account should now be able to subscribe to "arn:aws:sns:eu-west-1:299497370133:digitised-bag-notifications-workflow-staging". We should now be able to test that the subscription works and that the messages contain the necessary information.

**Comment by tomcrane** [comment]
Thanks @mgeerdsen 
I'll update our Terraform and test all that tomorrow.

We should be able to hook up the DDS `iiif-stage-new` environment so that stage-goobi notifications to that topic and up with created IIIF Manifests.

**Comment by tomcrane** [comment]
Will the production version of this topic be called `digitised-bag-notifications-workflow-prod`? Just making matching terraform to it for later use.

**Comment by mgeerdsen** [comment]
Indeed, the topic for production will be called `digitised-bag-notifications-workflow-prod`

**Comment by mgeerdsen** [comment]
To move this to production we will have to tag the stage workflow image for prod, merge #446 (already used for stage), terraform deploy it and use prepared sql statements to insert the new notification step after the current API call step.

**Comment by mgeerdsen** [comment]
SNS topic and permissions etc. have been added via terraform. Prod Goobi has been updated to include the necessary plugin.
We will update the workflows to include both the API call and SNS notification tomorrow.

**Comment by mgeerdsen** [comment]
and we actually did that, so this issue can be closed as well
